### PR TITLE
alwayspullimages admission controller: Test admission of containers when initcontainers are accepted

### PR DESCRIPTION
**Pull request by fntlnz**
Signed-off-by: Lorenzo Fontana <lo@linux.com>

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by fntlnz** [comment]
Is this ready to be approved?

**Comment by nikhita** [comment]
> Is this ready to be approved?

@fntlnz sorry that it's taking time to get this approved. Most approvers are usually pretty busy but they should get to this soon.

/assign @deads2k

**Comment by fntlnz** [comment]
I'm not sure if there's anyone else I can tag here :neutral_face:

**Comment by fntlnz** [comment]
@nikhita can you help me triaging this please?

File: plugin/pkg/admission/alwayspullimages/admission_test.go [hunk 1]
```diff
@@ -93,6 +93,31 @@ func TestValidate(t *testing.T) {
 	}
 }
 
+func TestValidateAdmittedInitContainers(t *testing.T) {
+	namespace := "test"
+	handler := &AlwaysPullImages{}
+	pod := api.Pod{
```
Comment (yue9944882) [inline plugin/pkg/admission/alwayspullimages/admission_test.go]: @fntlnz thx for contributing this. 

i suppose we can fold this test into `TestValidate`^^^ as one test case. wdyt?

Comment (fntlnz) [reply plugin/pkg/admission/alwayspullimages/admission_test.go]: yes makes sense, thanks for the suggestion

**Comment by fntlnz** [comment]
I think I'll just close this at this point, been open for half a year now and it's really just adding tests.

**Pull request closed without merging**